server/src/resolvers/MemberResolver.ts

```diff
@@ -32,7 +32,16 @@ export default class MemberResolver {
 		return user;
 	}
 	@Mutation(() => Member)
-	async signUp(@Args() { username, email, password }: SignUpArgs): Promise<Member> {
+	async signUp(
+		@Args() { username, email, password, confirmedPassword }: SignUpArgs
+	): Promise<Member> {
+		let existingEmail = (await MemberServices.findOneBy({ email })) as Member;
+		let existingUsername = (await MemberServices.findOneBy({ username })) as Member;
```

Comment:
why not use const here? Or if we are using let, create a modifiable variable like:
```
let verify = (await Member.Services.findOneBy({ email })) as Member;

if (verify) throw Error.Messages.EMAIL_ALREADY_REGISTERED_ERROR_MESSAGE;

verify = (await Member.Services.findOneBy({ username })) as Member;

if (verify) throw Error.Messages.USERNAME_ALREADY_REGISTERED_ERROR_MESSAGE;
```

Reply:
yes nice catch, i just will use a const